os/drivers/lcd/lcd_logo.c

```diff
@@ -0,0 +1,93 @@
+/****************************************************************************
+ *
+ * Copyright 2024 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <sys/types.h>
+#include "lcd_logo.h"
+#include <debug.h>
+
+extern const uint8_t lcd_logo_raw_data[LOGO_XRES * LOGO_YRES * 2];
+uint8_t *lcd_init_fullscreen_image = NULL;

```

Comment:
Is there any reason to add `fullscreen` word?
Will we support partial image later?